tests/core/arch-arm64/arm64stalker.c

```diff
@@ -0,0 +1,652 @@
+/*
+ * Copyright (C) 2009-2014 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com>
+ * Copyright (C) 2010-2013 Karl Trygve Kalleberg <karltk@boblycat.org>
```

Comment:
Please append yourself here and to all other copyright headers of files you added code to.